native/src/modules/common/renderHtml.js

```diff
@@ -85,7 +85,13 @@ const renderJS = (cacheDictionary: { [remoteUrl: string]: string }) => `
 `
 
 // language=HTML
-const renderHtml = (html: string, files: PageResourceCacheStateType, theme: ThemeType, language: string, resourceCacheUrl: string) => {
+const renderHtml = (
```

Comment:
Please avoid reformats. If this is due vscode then we somehow need to adjust our formatters.

Reply:
In fact I think this are all changes due to max line length so I think we don't have a problem. IntelliJ won't reformat these changes. I was playing with the idea of fixing this warnings for some time already as well :D

Reply:
Yes and yes - I tried to fix a linting error in my code and started from top of my linting errors
So I accidently solved the warnings (with manual autoformat). After solving my "real" error I decided to commit the solutions to the warnings also.
Sorry for the many changed files because of this